### PR TITLE
Add event 2018-04-05-mesh-workshop-trial-run

### PR DESCRIPTION
**Pull request by benhylau**
@MisterChalmers thanks for arranging venue, let me know if this looks okay :)

### CODE REVIEW

File: _posts/2018-04-05-mesh-workshop-trial-run.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Mesh Workshop Trial Run"
+slug: mesh-workshop-trial-run
+text: 'Trial run of the upcoming Building the Peer-to-Peer Internet workshop'
+location: HackLab.to
```
Comment (garrying) [inline _posts/2018-04-05-mesh-workshop-trial-run.md]: `HackLab.TO, 1266 Queen St`? (Looking at the previous events)

File: _posts/2018-04-05-mesh-workshop-trial-run.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+locationLink: https://www.openstreetmap.org/node/3522641496
+date: 2018-04-05
+startTime: '19:00'
+endTime: '21:00'
+---
+
+This is a trial run of our upcoming [Building the Peer-to-Peer Internet](https://tomesh.net/peer-to-peer-internet/) workshop series at the Toronto Public Library. During the two-hour session, we will be running the first module of the series in a group setting for the first time. The goal is to try out the format and identify friction points in the workshop, in preparation for the official workshop series.
+
+Everyone is welcome and no registration is required for this trial workshop, but resources are limited so please bring your laptop and a Raspberry Pi 2 or 3 if you have one. We hope you can join us at [{{ page.location }}]({{ page.locationLink }}), {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }}.
```
Comment (garrying) [inline _posts/2018-04-05-mesh-workshop-trial-run.md]: `{{ page.date | date: '%B %d, %Y' }}` -> `{{ page.date | date: '%B %-d, %Y' }}`
(Might just be me who is 🤔with leading 0's in dates)

**Review by garrying — COMMENTED** [review]
I think the title is fine, but had a small thought that this could have been titled a workshop preview

**Comment by benhylau** [comment]
@garrying thanks for the comments. Merged with changes.

**Pull request merged**